test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -33,8 +33,12 @@ class test_ucp_tag_xfer : public test_ucp_tag {
         enable_tag_mp_offload();
 
         if (RUNNING_ON_VALGRIND) {
-            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_SEG_SIZE", "8k"));
-            m_env.push_back(new ucs::scoped_setenv("UCX_TCP_RX_SEG_SIZE", "8k"));
+            // Alow using TM MP offload for messages with a size of at least
+            // 10000 bytes by setting HW TM segment size to 10 kB, since each
+            // packet in TM MP offload is MTU-size buffer (i.e., in most cases
+            // it is 4 kB segments)
+            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_SEG_SIZE", "10kb"));
```

Comment:
why is 10 better than 8?

Reply:
to reproduce the issue for 10000 size - it uses 2-segment Eager for this size